docs/getting-started/server/advanced-setup.md

```diff
@@ -26,6 +26,23 @@ without actually being charged.
    card number will need to be the correct number of characters).
 4. Buy premium features to your heart's content
 
+:::info
+
+Stripe has a [policy](https://support.stripe.com/questions/test-mode-subscription-data-retention) to
+**automatically cancel** test subscriptions after 90 days and then **delete** cancelled test
+subscriptions after 30 days. This can cause unexpected Billing behaviors for long-lived premium
+users/organizations in your local server after the subscription is removed.
+
+To correct this, you must re-subscribe the organization/user to a premium plan to create a new test
+subscription.
+
+1. From the Bitwarden Portal, remove the organization/user gateway information and set their plan to
```

Comment:
I believe Admin Portal is the proper name:

```suggestion
1. From the Admin Portal, remove the organization/user gateway information and set their plan to
```

Reply:
This could be outdated, but I did see there is an [open PR](https://github.com/bitwarden/contributing-docs/pull/123#issue-1717746050) renaming this to "Bitwarden Portal" in the docs. 

@trmartin4 Do you know which name is preferred going forward?

Reply:
I checked with Product, and it is a bit more nuanced than I thought.  The cloud version (which we use internally to manage our cloud-hosted instances) is called "Bitwarden Portal", but the same code running on a self-hosted instance is called the "System Management Portal".  It's the same code (albeit with some `isSelfHosted()` checks that disable some functionality like RBAC), but it has different names from a Product Portfolio perspective.

I got this clarification from Product.  Given that, I _think_ the portal referenced in this change would be the Bitwarden Portal.

To clarify this, I'm going to open another PR and add a callout on the contributing docs page to make sure that this is understood.

Reply:
Thanks for clarifying!